firedrake/plot.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+import numpy as np
+from ufl import cell
+from firedrake import Function, SpatialCoordinate
+
+
+def plot_mult(functions, num_points=100):
```

Comment:
`plot_multiple`?  Plus docstring.  I wonder if this can be a private function and the public `plot` interface can just check `if isinstance(function, Function):`.  What do you think?